oak-run/src/main/java/org/apache/jackrabbit/oak/plugins/tika/TextExtractor.java

```diff
@@ -79,6 +79,8 @@ class TextExtractor implements Closeable {
     private boolean initialized;
     private BinaryStats stats;
     private boolean closed;
+    private boolean linkageErrorFound = false;
```

Comment:
There is no need to set them to false: they are false by default.

Reply:
done